inst/htmlwidgets/gauge.js

```diff
@@ -41,26 +42,25 @@ HTMLWidgets.widget({
         // justgage config
         var config = {
           id: el.id,
-          title: " ",
-          valueFontColor: "gray",
           value: x.value,
           min: x.min,
           max: x.max,
-          titlePosition: "below",
           relativeGaugeSize: true,
           formatNumber: true,
           humanFriendly: x.humanFriendly,
           humanFriendlyDecimal: x.humanFriendlyDecimal,
           customSectors: x.customSectors
         };
 
-        // add symbol if specified
-        if (x.symbol !== null)
-          config.symbol = x.symbol;
-
-        // add label if specifed
-        if (x.label !== null)
-          config.label = x.label;
+        // Add these props to the config if specified
+        var optional_props = [
+          "symbol", "label", "gaugeColor", "valueFontColor",
+          "valueFontFamily", "labelFontColor", "labelFontFamily"
+        ]
+        for (var i = 0; i < optional_props.length; i++) {
+          var prop = optional_props[i];
+          if (x[prop]) config[prop] = x[prop];
```

Comment:
Do we need to copy over falsy values for `x[prop]`? If so, maybe it should be `if (x.hasOwnProperty(prop))`.

Reply:
Good point. I don't think it's relevant, but that seems the safer/better thing to do.

Reply:
Oh, actually this is important for `symbol` -- the current behavior is what we want (don't copy over falsy values).

Reply:
What about when `label` is `""`, or if `symbol` is 0?

I'm thinking we should keep the explicit `null` check logic for `symbol` and `label`, but use this `for` loop for the others. The concern is that there will be unintended consequences of using the falsy check.